Jenkinsfile

```diff
@@ -1,10 +1,24 @@
 //https://github.com/feedhenry/fh-pipeline-library
+import org.feedhenry.Utils
 
 fhBuildNode {
+
+    def utils = new Utils()
+
     stage('Install Dependencies') {
         npmInstall {}
     }
 
+    stage('Test') {
+        gruntCmd {
+            cmd = 'format'
+        }
+        if(utils.gitRepoIsDirty()) {
```

Comment:
In this block here, what's the difference between `print` and the `echo` pipeline step? I see there's an `error` step also that takes an optional message, would that be better than `sh('exit 1')` or is there some reason we shouldn't use it? Given that it takes a message, it could maybe replace both lines here. WDYT?

https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-error-code-error-signal